src/store/TranscriptionsStore.js

```diff
@@ -529,6 +545,7 @@ const TranscriptionsStore = types.model('TranscriptionsStore', {
     setParsedExtracts,
     setTextObject,
     setTranscription: (transcription) => undoManager.withoutUndo(() => setTranscription(transcription)),
+    test,
```

Comment:
Is this meant to be here? It's crashing the app when I try to compile: `ReferenceError: test is not defined`

Reply:
Ah no. Have to clean that up